### ISSUE TITLE
Pinned tabs not reloading

### ISSUE THREAD

**Issue by harryray33** [hackademix/noscript#300]
The latest version of Noscript 11.4.22 is causing any tabs that are pinned to not reload on a restart.
On restart, there is no URL in the address bar of the pinned tab.

I'm using FF 88.0.1

**Comment by hackademix** [comment]
>  Noscript 11.4.22 

The changes between 11.4.21 and 11.4.22 are very unlikely to affect tab loading, unless your pinned tabs are data: or file:// URLs.

1. Is that the case?
2. Does [downgrading to 11.4.21](https://noscript.net/feed) actually help (and upgrading again break those tabs again)?

> I'm using FF 88.0.1

That's an obsolete, unsafe and unsupported version. I cannot reproduce on the oldest supported Firefox ESR (102). 
Is there any compelling reason for you not to upgrade?

**Comment by harryray33** [comment]
Thanks, yes, I tried it a few times and there isn't a problem with 11.4.21.
Possibly a combination of Noscript and a different extension that could be causing this? I'm running the portable version of FF, but I shouldn't think that makes ant difference.

The pinned tab is for gmail...https://mail.google.com/mail/u/0/#inbox/.............

I'll do a portable install of the most recent FF and see if it still happens. 

Update: 
I also noticed that Noscript seems to be slowing down, and sometimes not loading Youtube videos. I've experimented, and it does appear to be caused by the latest version.

**Comment by hackademix** [comment]
>  Possibly a combination of Noscript and a different extension that could be causing this? 

Yes, it's a possibility. Does disabling all the other extensions help?

> The pinned tab is for gmail...https://mail.google.com/mail/u/0/#inbox/.............

I've got a pinned GMail tab in my main Firefox profile as well, and did not notice anything weird.

> I'll do a portable install of the most recent FF and see if it still happens.

Thanks.

**Comment by harryray33** [comment]
It appears to be uBlock origin that's causing the problem...

**Comment by harryray33** [comment]
I'll put up an issue on the uBlock site, but is there anything that can be done at your end?

**Comment by hackademix** [comment]
> I'll put up an issue on the uBlock site, but is there anything that can be done at your end?

1. Have you got any non-default uBlock setting (and if that's the case, could you share them by using the "Backup to file" feature)?
2. Does the browser console (ctrl+shift+J) and/or the content console (ctrl+shift+K on the broken pinned tab) show any NoScript-related message?
3. Is uBlock apparently blocking anything on the pinned tab?

**Comment by harryray33** [comment]
As far as I can tell, there's nothing related to NoScript in the console.

As far as I recall, I haven't changed any default settings in the last few months. My version of uBlock was updated about 3 weeks ago, so it's been working with NoScript until the latest update.
Would it help if I sent you the uBlock settings file?

**Comment by hackademix** [comment]
> Would it help if I sent you the uBlock settings file?

Definitely. Also, did you have a change to try on a clean profile with a fresh uBlock instance and/or a recent Firefox?

**Comment by harryray33** [comment]
I've just had another look, and I've noticed errors starting with [NoScript]:0 Prompt Hook installation..........js:32:13

Where should I send the uBlock settings file to?

**Comment by hackademix** [comment]
> Where should I send the uBlock settings file to?

The email address on the top right of https://maone.net, thanks.

**Comment by harryray33** [comment]
Thanks, I'll send that and the full error message.

I re-installed uBlock, with all it's default settings and a minimal filter list, but still had the same problem.
I can't work out why this should only affect a pinned tab...

**Comment by VL4DST3R** [comment]
Came here after investigating this issue as well on Waterfox G3, works well with UBO on 11.4.21 but not .22. Was this reported to their bugtracker? If so, where? i can't seem to find anything.

I also found [this ](https://forums.informaction.com/viewtopic.php?t=20901)old noscript forum thread that describes the same issue, even including the workaround that ive also been using since this started happening of "_Clicking the "Back" button, followed by clicking the "Forward" button, brings you back [...]_". Maybe it's related?

**Comment by TFWol** [comment]
Noticed this issue too. It happens as soon as I click update on NoScript (I update my add-ons one at a time).

> "_Clicking the "Back" button, followed by clicking the "Forward" button, brings you back [...]_". Maybe it's related?

I've had to do this as well.

**Comment by harryray33** [comment]
Tried installing the new version of Noscript, but still having the same problem with my pinned tab.
